portfolio/src/main/webapp/script.js

```diff
@@ -132,3 +132,99 @@ async function formLoad() {
     statsListElement.style.color = rate_value;
   });
 }
+
+async function storeData() {
+  fetch('/store-data').then(response => response.json()).then((tasks) => {
+    const taskListElement = document.getElementById('form-data');
+    tasks.forEach((task) => {
+      taskListElement.appendChild(createTaskElement(task));
+    })
+  });
+}
+
+function createMap() {
+  const directionsRenderer = new google.maps.DirectionsRenderer();
+  const directionsService = new google.maps.DirectionsService();
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 55.752779, lng: 37.621588},
+      zoom: 14,
+      clickableIcons: true,
+      backgroundColor: "#red"
+      });
+  target = new google.maps.Marker({
+    position: {lat: 55.752779, lng: 37.621588},
+    map,
+    title: "There is me!"
+  });
+  var pos = {lat: 55.752779, lng: 37.621588};
+  infoWindow = new google.maps.InfoWindow;
+  if (navigator.geolocation) {
+        navigator.geolocation.getCurrentPosition(function(position) {
+        pos = {
+          lat: position.coords.latitude,
+          lng: position.coords.longitude
+          };
+            infoWindow.setPosition(pos);
+            infoWindow.setContent('Location found.');
+            infoWindow.open(map);
+            map.setCenter(pos);
+        }, function() {
+            handleLocationError(true, infoWindow, map.getCenter(), map);
+          });
+  } else {
+          // Browser doesn't support Geolocation
+          handleLocationError(false, infoWindow, map.getCenter(), map);
+  }
```

Comment:
The intentation here looks a bit funny. Other code is indented by 2 spaces but this is indented by 6. I would expect this to look like:

```javascript
if (navigator.geolocation) {
  navigator.geolocation.getCurrentPosition(function(position) {
    pos = {
      lat: position.coords.latitude,
      lng: position.coords.longitude
    };
    infoWindow.setPosition(pos);
    infoWindow.setContent('Location found.');
    infoWindow.open(map);
    map.setCenter(pos);
  }, function() {
    handleLocationError(true, infoWindow, map.getCenter(), map);
  });
} else {
  // Browser doesn't support Geolocation
  handleLocationError(false, infoWindow, map.getCenter(), map);
}
```